src/functionalTest/java/uk/gov/hmcts/dts/fact/admin/AdminCourtFacilityEndpointTest.java

```diff
@@ -0,0 +1,164 @@
+package uk.gov.hmcts.dts.fact.admin;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.dts.fact.model.admin.Facility;
+import uk.gov.hmcts.dts.fact.util.AdminFunctionalTestBase;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.http.HttpHeaders.AUTHORIZATION;
+import static org.springframework.http.HttpStatus.*;
+import static uk.gov.hmcts.dts.fact.util.TestUtil.BEARER;
+import static uk.gov.hmcts.dts.fact.util.TestUtil.objectMapper;
+
+@ExtendWith(SpringExtension.class)
+public class AdminCourtFacilityEndpointTest extends AdminFunctionalTestBase {
+
+    private static final String ADMIN_COURTS_ENDPOINT = "/admin/courts/";
+    private static final String FACILITIES_PATH = "/facilities";
+    private static final String AYLESBURY_COUNTY_COURT_AND_FAMILY_COURT_SLUG = "aylesbury-county-court-and-family-court";
+    private static final String GREENWICH_MAGISTRATES_COURT_SLUG = "greenwich-magistrate-court";
+
+
+    private static final String AYLESBURY_COURT_FACILITIES_PATH = ADMIN_COURTS_ENDPOINT + AYLESBURY_COUNTY_COURT_AND_FAMILY_COURT_SLUG
+        + FACILITIES_PATH;
+    private static final String COURT_NOT_FIND_PATH = ADMIN_COURTS_ENDPOINT + GREENWICH_MAGISTRATES_COURT_SLUG
+        + FACILITIES_PATH;
+
+    private static final String TEST_FACILITY_NAME1 = "Lift";
+    private static final String TEST_FACILITY_NAME2 = "Refreshments";
+    private static final String TEST_FACILITY_DESCRIPTION = "TEST";
+    private static final String TEST_FACILITY_DESCRIPTION_CY = "TESTCY";
+
+
+    /************************************************************* GET request tests section. ***************************************************************/
+    @Test
+    public void getFacilitiesForTheCourt() {
```

Comment:
@ashwini-mv this is likely to cause a PMD issue. You can run "gradlew build" before committing the changes to check for this. Essentially PMD will assume a method starting with a "get" keyword is a getter and must always return a value. You need to rename this method slightly.